src/mongo/db/auth/oidc/oidc_server_parameters.cpp

```diff
@@ -75,7 +75,7 @@ bool operator<(const IssuerAudiencePair& lhs, const IssuerAudiencePair& rhs) noe
 
 template <typename R>
 requires std::ranges::range<R> && std::is_same_v<std::ranges::range_value_t<R>, std::size_t>
-str::stream errorMsgHeader(const R& indexes) {
+    str::stream errorMsgHeader(const R& indexes) {
```

Comment:
The additional indent is clearly incorrect. If the formatter provides a mechanism for excluding a particular code chunk (e.g. something like `// format-off` tag), probably we should use it here. If no such mechanism exists, then let's leave this change as is.